src/leap/bitmask/core/mail_services.py

```diff
@@ -0,0 +1,668 @@
+"""
+Mail services.
+
+This is quite moving work still.
+This should be moved to the different packages when it stabilizes.
+"""
+import json
+import os
+from glob import glob
+from collections import defaultdict
+from collections import namedtuple
+
+from twisted.application import service
+from twisted.internet import defer
+from twisted.python import log
+
+from leap.bonafide import config
+from leap.common.service_hooks import HookableService
+from leap.keymanager import KeyManager, openpgp
+from leap.keymanager.errors import KeyNotFound
+from leap.soledad.client.api import Soledad
+from leap.mail.constants import INBOX_NAME
+from leap.mail.mail import Account
+from leap.mail.imap.service import imap
+from leap.mail.incoming.service import IncomingMail, INCOMING_CHECK_PERIOD
+from leap.mail import smtp
+
+from leap.bitmask.core.uuid_map import UserMap
+
+
+class Container(object):
+
+    def __init__(self):
+        self._instances = defaultdict(None)
+
+    def get_instance(self, key):
+        return self._instances.get(key, None)
+
+
+class ImproperlyConfigured(Exception):
+    pass
+
+
+def get_all_soledad_uuids():
```

Comment:
I forgot what this what intended for. deleting.
it probably is no longer relevant with the uuid_map implementation.